shim-deps/cloudera/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2023, NVIDIA CORPORATION.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.nvidia</groupId>
+        <artifactId>rapids-4-spark-parent</artifactId>
+        <version>23.10.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>rapids-4-spark-cdh-bom</artifactId>
```

Comment:
what does bom stand for?

Reply:
Usually it is bill of materials.

https://www.investopedia.com/terms/b/bill-of-materials.asp

Reply:
I can rename if needed. My original intention was to use a pure [BOM pattern](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms) I found recommended somewhere on stackoverflow. However, `dependencyManagement` does not support `exclusions` used in the CDH context, and even if it would, we still need to do some consolidation to avoid repeating `<dependencies>`. I found that the latter makes the former redundant. So the definition of a BOM in our project is not really the same what the Maven doc describes.

Reply:
its fine to leave as it is